PROGRESS.md

```diff
@@ -0,0 +1,113 @@
+Setup-
```

Comment:
Useful readme with clear start up instructions and clear indication of progress and where you ran into issues.